go.mod

```diff
@@ -35,13 +35,16 @@ require (
 	k8s.io/api v0.26.2
 	k8s.io/apiextensions-apiserver v0.26.2
 	k8s.io/apimachinery v0.26.2
-	k8s.io/client-go v1.5.2
+	k8s.io/client-go v12.0.0+incompatible
```

Comment:
This feels like a potentially dangerous upgrade?  What else uses client-go and what is incompatible about this version?

Reply:
I didnt manually change anything mod/vendor....It's auto updated `go mod tidy && go mod vendor && go mod verify` 

that said it's safe since this v0.26.1 is already pinned [here](https://github.com/openshift-kni/cnf-features-deploy/blob/0c55894ab9bedad180a5c347c81cee2252278026/go.mod#L150) (v1.5.2 didnt mean much)